### PR TITLE
Change parameter object keybind from `p` to `a`

### PR DESCRIPTION
**Pull request by sudormrfbin**
#1627 requires `p` for paragraph textobject which was being used by parameter. This PR replaces parameter's `p` with `a` which can be thought of as "argument", and replaces all instances of "parameter" with "argument" for consistent naming.

### CODE REVIEW

**Comment by archseer** [comment]
Can you rebase? Fixed lints on master

File: book/src/keymap.md [hunk 1]
```diff
@@ -268,8 +268,8 @@ Mappings in the style of [vim-unimpaired](https://github.com/tpope/vim-unimpaire
 | `[f`     | Go to previous function (**TS**)             | `goto_prev_function`  |
 | `]c`     | Go to next class (**TS**)                    | `goto_next_class`     |
 | `[c`     | Go to previous class (**TS**)                | `goto_prev_class`     |
-| `]p`     | Go to next parameter (**TS**)                | `goto_next_parameter` |
-| `[p`     | Go to previous parameter (**TS**)            | `goto_prev_parameter` |
+| `]a`     | Go to next argument (**TS**)                 | `goto_next_parameter` |
+| `[a`     | Go to previous argument (**TS**)             | `goto_prev_parameter` |
```
Comment (anthonyjclark) [inline book/src/keymap.md:272]:
```suggestion
| `]a`     | Go to next argument (**TS**)                 | `goto_next_argument` |
| `[a`     | Go to previous argument (**TS**)             | `goto_prev_argument` |
```

**Comment by anthonyjclark** [comment]
Should the docs still use "argument (parameter)" somewhere so that it is easy find in a search?

**Comment by archseer** [comment]
Could we keep parameter and use `P` for paragraph?

**Comment by sudormrfbin** [comment]
Paragraph movement is something I use fairly a lot, and I would say the same goes for most people. `]P` takes three keypresses which I think is too many for a common operation.

**Comment by archseer** [comment]
How about keeping the same name for the functions and query scopes, but using `a` for the binding (and "argument (parameter)" in the infobox)?

The highlight scopes already use `@variable.parameter` for these so it's a bit confusing if the textobject scopes are different.

**Review by archseer — APPROVED** [review]

**Pull request merged**